tools/iothub-explorer/iothub-explorer.js

```diff
@@ -165,6 +165,28 @@ else if (command === 'create') {
     }
   });
 }
+else if (command === 'import'){
+  if(!arg1) inputError('Import file not specified');
+  console.log(colorsTmpl('\n{green}Check if import file exists{/green}'));
```

Comment:
We don't want extraneous output in the "happy path" case.  Remove the log lines about checking for the import file, leaving only the inputError(...) when the import file doesn't exist.